src/components/InputForm.js

```diff
@@ -1,14 +1,11 @@
-import React from "react";
-class InputForm extends React.Component{
-    render(){
-        return (
-            <form>
-                <input type="text" name="title" placeholder="Enter NAme of Book"/>
-                <input type="text" name="author" placeholder="Enter Name of AUthor" />
+import React from 'react';
 
+const InputForm = () => (
+  <form>
+    <input type="text" name="title" placeholder="Enter NAme of Book" />
+    <input type="text" name="author" placeholder="Enter Name of AUthor" />
 
```

Comment:
- [ ] [REQUIRED] Please, make sure that you have a submit button with label "Add Book" as mentioned in project requierments.